app/views/purchases/index.html.erb

```diff
@@ -0,0 +1,78 @@
+<div class="blue">
+  <div class="container-sm">
+    <div class="clase-header2 blue d-flex justify-between align-end">
+      <a href="<%= categories_path %>" class="back-button"><i class="fa-solid fa-arrow-left" style="color: #ffffff;"></i></a>
+      <h1 class="category-title"><%= @category.name %></h1>
+      <a href="<%= categories_path %>" class="btn-login blue"><i class="fa-solid fa-magnifying-glass" style="color: #ffffff;"></i></a>
+    </div>
+    <div class="d-flex justify-between gap-10">
+            <div class="w-1-3">
+
+            </div>
+
+            <div class="small-div w-1-3 d-flex flex-column align-center justify-end most-recent-container ">
+              <p class="text">Most recent</p>
+              <p class="dot">•</p>
+            </div>
+            <div class="small-div w-1-3 d-flex flex-column align-center justify-end most-ancient-container">
+              <p class="text">Most ancient</p>
+              <p class="dot">•</p>
+            </div>
+    </div>
+  </div>
+</div>
+
+<div class="container-sm">
+  <div class="total-amount d-flex justify-between align-center">
+    <h2>Total Amount:</h2>
+    <p>$<%= @category.category_total %></p>
+  </div>
+</div>
+
+<hr>
+
+<div class="container-sm">
+  <h2>Transactions:</h2>
```

Comment:
- [ **OPTIONAL** ] To correctly match the organization of the page and make it be significantly, I think it would be better if you replace the **Transactions** title where you put of the `<%= @category.name %>` and vice versa. The title of the will be at the top then the `<%= @category.name %>` at his expected place.
![Screenshot from 2023-10-19 06-07-27](https://github.com/ClaudiaRojasSoto/BudgetApp/assets/83042742/f90fcd0f-864f-47f0-a708-ef5b9e6f2726)

Reply:
Done!, thank you @tresorsawasawa